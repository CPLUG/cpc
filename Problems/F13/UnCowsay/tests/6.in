 _________________________________________________________________________ 
/ It is undignified for a woman to play servant to a man who is not hers. \
\                 -- Spock, "Amok Time", stardate 3372.7                  /
 ------------------------------------------------------------------------- 
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
