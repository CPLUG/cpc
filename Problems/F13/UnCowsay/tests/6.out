It is undignified for a woman to play servant to a man who is not hers.
                -- Spock, "Amok Time", stardate 3372.7                 
